QModManager/Utility/Config.cs

```diff
@@ -62,8 +62,14 @@ private static void Save()
         {
             try
             {
-                string text = JsonConvert.SerializeObject(Cfg, Formatting.Indented);
-                File.WriteAllText(ConfigPath, text);
+                var serializer = new JsonSerializer
+                {
+                    NullValueHandling = NullValueHandling.Ignore
```

Comment:
```cs
MissingMemberHandling = MissingMemberHandling.Ignore
```
?

Reply:
[MissingMemberHandling](https://www.newtonsoft.com/json/help/html/P_Newtonsoft_Json_JsonSerializer_MissingMemberHandling.htm) is for _deserialization_.  
It has no impact here for writing the config.